requirements.txt

```diff
@@ -2,3 +2,4 @@
 # Useful for dev/test jobs caches
 # Must be kept in sync with setup.py
 torch == 2.2.0 # This is what is installed in CI today
+safetensors
```

Comment:
we really only need to enforce this for unit testing, in general unless you want to load safetensors checkpoints you shouldn't need it